src/ua-parser.js

```diff
@@ -898,6 +901,24 @@
         ]
     };
 
+    var electronUA = function () {
+        // Renderer process
+        if (typeof window !== 'undefined' && typeof window.process === 'object' && window.process.type === 'renderer') {
+            return 'electron';
+        }
+
+        // Main process
+        if (typeof process !== 'undefined' && typeof process.versions === 'object' && !!process.versions.electron) {
+            return 'electron'+'/'+process.versions.electron;
+        }
+
+        // Detect the user agent when the `nodeIntegration` option is set to true
+        if (typeof navigator === 'object' && typeof navigator.userAgent === 'string' && navigator.userAgent.indexOf('Electron') >= 0) {
+            return 'electron';
+        }
+
+        return false;
```

Comment:
@andresgallego I think here would be better to just return undefined `return;` to avoid mixing boolean and strings as return values for the same function.